pkg/agent/interfacestore/interface_cache.go

```diff
@@ -227,13 +243,19 @@ func podIndexFunc(obj interface{}) ([]string, error) {
 	return []string{k8s.NamespacedName(interfaceConfig.PodNamespace, interfaceConfig.PodName)}, nil
 }
 
+func interfaceIPIndexFunc(obj interface{}) ([]string, error) {
+	interfaceConfig := obj.(*InterfaceConfig)
+	return []string{interfaceConfig.IP.String()}, nil
```

Comment:
Not all interfaces have IP, better to check whether IP is nil before indexing it to save some memory?

Reply:
Yes, I missed it.